website/app/assets/javascript/components/component-examples.js

```diff
@@ -1,9 +1,9 @@
-import { iframeResizer } from "iframe-resizer";
+import { initialize } from "@open-iframe-resizer/core";
 
 export default function initComponentExamples() {
   // Resize component iFrames using https://www.npmjs.com/package/iframe-resizer
```

Comment:
Should ideally update this comment as well!

```suggestion
  // Resize component iFrames using https://www.npmjs.com/package/@open-iframe-resizer/core
```